### DISCUSSION TITLE
Add mixed variable many objective optimization benchmark.

### DISCUSSION THREAD

**Discussion by muazhari** [anyoptimization/pymoo#615] [Ideas]
In [this](https://pymoo.org/problems/definition.html), I don't see any provided problem for mixed variable many-objective optimization ready to be benchmarked by [this](https://pymoo.org/misc/indicators.html). Can you add it?

**Comment by blankjul** [comment]
Mixed-variable optimization is treated as an application/tutorial in pymoo as none of the algorithms have been direclty designed for this. 
This is why we decided to keep the documentation separate as well.

**Reply by muazhari** [reply]
![image](https://github.com/user-attachments/assets/e96cc05b-ca9b-4cd9-a74d-c94b6c928abe)

Do you mean the [mixed variable example](https://pymoo.org/customization/mixed.html) is untrue?
If so, then you should correct the docs to be more specific.

**Reply by blankjul** [reply]
Can you push a PR adding the comment that would have helped you to understand the capabilities better?

**Reply by muazhari** [reply]
I think it should not be me to contribute to the docs. I am just being confused about the information in the docs by not being explicit. If I am confused and lack understanding, I should not add/reduce the information in the docs right?
